test/integration-test/bpf/kconfig.bpf.c

```diff
@@ -0,0 +1,48 @@
+// clang-format off
+#include <vmlinux.h>
+#include <bpf/bpf_helpers.h>
+// clang-format on
+
+// Test char (1 byte alignment)
+// CONFIG_BPF=y => 1
+extern unsigned char CONFIG_BPF __kconfig;
+// Test short (2 byte alignment)
+// CONFIG_PANIC_TIMEOUT=0 => 0
+extern unsigned short CONFIG_PANIC_TIMEOUT __kconfig;
+// Test int (4 byte alignment)
+// CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=120
+extern unsigned int CONFIG_DEFAULT_HUNG_TASK_TIMEOUT __kconfig;
+// Test long (8 byte alignment)
+// CONFIG_BPF_JIT=y => 1
+extern unsigned long CONFIG_BPF_JIT __kconfig;
+// CONFIG_DEFAULT_HOSTNAME
+extern char CONFIG_DEFAULT_HOSTNAME[] __kconfig;
+
+SEC("xdp")
+int kconfig(struct xdp_md *ctx) {
+  if (CONFIG_BPF != 1) {
+    return XDP_DROP;
+  }
+
+  if (CONFIG_PANIC_TIMEOUT != 0) {
+    return XDP_DROP;
+  }
+
+  if (CONFIG_DEFAULT_HUNG_TASK_TIMEOUT != 120) {
```

Comment:
is this getting tested anywhere? that the thing returns DROP if the values are wrong/corrupted?